zerver/views/webhooks/heroku.py

```diff
@@ -0,0 +1,25 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from typing import Any, Callable, Iterable, Optional, Tuple
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from six import text_type
```

Comment:
Import from typing Text instead of text_type. We just changed the typing in this directory.